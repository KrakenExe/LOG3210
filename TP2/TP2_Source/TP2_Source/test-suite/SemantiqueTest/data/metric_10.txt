Declaration
num a;
bool b;
EndDeclaration

a = 1;

while (a + 1 * 3 < 10 + a) {
  if (false) {
    b = -a < 1;
  }
  else {
    b = !(a > 1);
  }
}