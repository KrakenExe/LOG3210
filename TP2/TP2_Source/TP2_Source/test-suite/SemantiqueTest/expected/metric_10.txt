{VAR:2, WHILE:1, IF:1, FOR:0, OP:8}